package.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "@eclipse-cdt-cloud/vscode-ui-components",
+    "displayName": "VSCode UI Components",
+    "description": "VSCode UI Components for React",
+    "version": "0.0.1",
+    "publisher": "eclipse-cdt",
+    "license": "MIT",
+    "repository": "https://github.com/eclipse-cdt-cloud/vscode-ui-components",
+    "qna": "https://github.com/eclipse-cdt-cloud/vscode-ui-components/issues",
+    "main": "./lib/index.js",
+    "types": "./lib/index.d.ts",
+    "files": [
+        "lib"
+    ],
+    "engines": {
+        "node": ">=20"
+    },
+    "scripts": {
+        "prepare": "yarn clean && yarn build",
+        "clean": "rimraf ./lib ./node_modules/.tmp",
+        "build": "tsc && yarn lint",
+        "watch": "tsc -w",
+        "lint": "eslint ."
+    },
+    "dependencies": {
+        "@floating-ui/react": "^0.26.17",
+        "@vscode/codicons": "0.0.20",
+        "@vscode/webview-ui-toolkit": "^1.4.0",
```

Comment:
Interesting, to what extent are we still using this toolkit? It has been declared deprecated

Reply:
I will update this PR (and the dependencies) again after merging https://github.com/eclipse-cdt-cloud/vscode-peripheral-inspector/pull/46, therefore some of the dependencies may be removed. Thanks for highlighting it.

Reply:
OK, I see. Is closing the peripheral inspector PR a hard dependency? Or could we swap the order if it would block you here?

Reply:
Any order is fine, I will fix this PR and then we can either merge or wait.

Reply:
OK, thanks. Let's see then which lands first.